src/System.Transactions.Local/src/System/Transactions/DurableEnlistmentState.cs

```diff
@@ -11,128 +11,29 @@ namespace System.Transactions
     internal abstract class DurableEnlistmentState : EnlistmentState
     {
         // Double-checked locking pattern requires volatile for read/write synchronization
```

Comment:
I removed this comment in few places but missed removing the comment here and there are other places too which I will remove. Actually found a few more places for using Lazy init construct, while searching for this comment. Will make those changes and commit them.